Client/TabManagement/TabManagerImplementation.swift

```diff
@@ -467,3 +475,75 @@ class TabManagerImplementation: LegacyTabManager, Notifiable {
         }
     }
 }
+
+extension TabManagerImplementation {
+    // Ecosia: Get Sites from places
+    private func fetchDBSites() async -> [Site] {
```

Comment:
I would suggest a more descriptive name what this method does. E.g. `fetchLastVisitedSitesFromDB(count: Int)` so it's transparent to the caller